app/websocket/app/main/model_manager.py

```diff
@@ -0,0 +1,195 @@
+import syft as sy
+
+import pickle
+import os
+
+from .persistence.models import db, MLModel
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+
+
+model_cache = dict()
+
+
+def clear_cache():
+    """Clears the cache.
+
+    """
+    model_cache = dict()
+
+
+def is_model_in_cache(model_id: str):
+    """Checks if the given model_id is present in cache.
+    
+    Args:
+        model_id (str): Unique id representing the model. 
+
+    Returns:
+        True is present, else False.
+
+    """
+
+    return model_id in model_cache
+
+
+def get_model_from_cache(model_id: str):
+    """Checks the cache for a model. If model not found, returns None.
+    
+    Args:
+        model_id (str): Unique id representing the model. 
+
+    Returns:
+        An encoded model, else returns None.
+
+    """
+
+    return model_cache.get(model_id)
+
+
+def save_model_to_cache(serialized_model: bytes, model_id: str):
+    """Saves the model to cache. Will fail if a model with same id already exists.
```

Comment:
The method will not fail, you could say "Saves the model to cache. Nothing happens if a model with the same id already exists."